applications/nrf5340_audio/tools/buildprog/buildprog.py

```diff
@@ -113,13 +113,17 @@ def __build_cmd_get(core: Core, device: AudioDevice, build: BuildType, pristine,
             device_flag += " -DCONFIG_AUDIO_DFU=2"
         if options.min_b0n:
             device_flag += " -DCONFIG_B0N_MINIMAL=y"
+
+        if options.controller == Controller.acs_nrf53:
+            device_flag += " -DCONFIG_BT_LL_ACS_NRF53=y"
+            device_flag += " -DCONFIG_NCS_INCLUDE_RPMSG_CHILD_IMAGE=n"
+
         if options.controller == Controller.sdc:
-            device_flag += " -DCONFIG_BT_LL_ACS_NRF53=n"
             if not child_image:
                 device_flag += " -DCONFIG_NCS_INCLUDE_RPMSG_CHILD_IMAGE=n"
 
         if options.nrf21540:
-            device_flag += " -DSHIELD=nrf21540ek_fwd"
+            device_flag += " -DSHIELD=nrf21540ek_fwd -Dhci_ipc_SHIELD=nrf21540ek"
```

Comment:
We would need to check the controller typeto decide how we should involve the support for nRF21504.